status-im/status-react
# feat: trigger swap transactions

fixes #20381 
fixes #20344 

### Summary

This PR integrates swaps transactions actually triggering after user confirms the transaction. Also adds proper toasts notifications as per designs.

https://github.com/user-attachments/assets/ae5b0be9-575c-437c-b57e-b2c88a43cf13

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Login
- Go to wallet
- Select an account
- Select Swap option
- Select a ERC20 token on Select asset to pay screen
- Enter a valid amount
- Wait for swap proposal to appear
- Tap on "Review swap"
- Slide to execute the transction
- Enter password
- Swap modal should be dismissed and toast should be shown
- Wait for the swap transaction to be confirmed / failed and proper toasts are shown 

status: ready
